activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -63,4 +63,12 @@
       <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/${jackson-databind-version}</bundle>
       <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson-version}</bundle>
     </feature>
+
+    <!-- JMS1.1 and JMS2.0 support via the same Geronimo JMS 2 bundle  -->
+    <!-- Ideal since all JMS code shares the same class definitions    -->
+    <!-- Fortunately, JMS2.0 is fully backward compatible with JMS 1.1 -->
+    <feature name="activemq-combined-jms-spec">
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">wrap:mvn:org.apache.geronimo.specs/geronimo-jms_2.0_spec/1.0-alpha-2$overwrite=merge&amp;Export-Package=javax.jms;version=1.1,javax.jms;version=2.0</bundle>
```

Comment:
Instead of rewrapping, why not simply installing the two spec versions (1.1 and 2.0) ?